src/components/Calculator.js

```diff
@@ -1,31 +1,43 @@
-import React from 'react';
+/* eslint-disable max-len */
```

Comment:
- [ ] Kindly fix all the disabled linters because linters help us write consistent and cleaner code